DESCRIPTION

```diff
@@ -16,6 +16,8 @@ LazyData: TRUE
 Imports:
     dplyr,
     stringr,
+    stringi,
```

Comment:
I see where you are importing a function from this package but I don't actually see where the function is used; so, do we need to add it in the DESCRIPTION file? We are actually already requiring it because stringr imports it so it is not a big deal if we leave this line I just don't see a reason.